README.md

```diff
@@ -1,4 +1,4 @@
-# More Ores
+# More Ores [![Build status](https://github.com/minetest-mods/moreores/workflows/build/badge.svg)](https://github.com/minetest-mods/moreores/actions)
```

Comment:
I don't think a build badge in the README is relevant these days, as GitHub already displays one next to the repository name (once this PR is merged).